packages/x-components/src/__stubs__/suggestions-stubs.factory.ts

```diff
@@ -58,7 +58,7 @@ export function createSuggestionStub(query: string, suggestion?: Partial<Suggest
     query,
     totalResults: 10,
     results: [],
-    modelName: 'Suggestion',
+    modelName: 'QuerySuggestion',
```

Comment:
This file is getting [errors](https://github.com/empathyco/x/pull/150/checks?check_run_id=3599561851#step:6:3131) regarding the ModelName.

Also we didn't update the `getSuggestionsStub` method as in the other modules:

```javascript
/**
 * Creates a list of {@link Suggestions} of the given length.
 *
 * @param length - The length of the list to create.
 * @returns A list of simple {@link Suggestions}s.
 */
export function getSuggestionsStub(length = 3): Suggestion[] {
  return Array.from({ length }, (_, index) => createSuggestionStub(index));
}
```

Reply:
If we do this change, we need to change in a lot of files (sorry for that), what do you think?